src/Sulu/Bundle/DocumentManagerBundle/DependencyInjection/SuluDocumentManagerExtension.php

```diff
@@ -101,6 +101,7 @@ public function prepend(ContainerBuilder $container)
                     'exception' => [
                         'codes' => [
                             'Sulu\Component\DocumentManager\Exception\DocumentNotFoundException' => 404,
+                            'Sulu\Component\DocumentManager\Exception\VersionNotFoundException' => 404,
```

Comment:
if we want use that in SuluArticleBundle we have to introduce a HOTFIX for that.